torch/overrides.py

```diff
@@ -1055,13 +1110,13 @@ def handle_torch_function(
 
     Returns
     -------
-    Result from calling `implementation()` or an `__torch_function__`
-    method, as appropriate.
+    object
+        Result from calling ``implementation`` or an ``__torch_function__``
+        method, as appropriate.
 
     Raises
     ------
     TypeError : if no implementation is found.
```

Comment:
I think the little sample code for all the places in our codebase where we call it manually would be very instructive here.